meveo-ejb/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I thought we agreed to not rename meveo-ejb, and keep meveo-admin-ejbs

Reply:
please check the next comment.